parts/configure-swarmmode-cluster.sh

```diff
@@ -150,9 +151,13 @@ echo "Installing and configuring Docker"
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh

```

Comment:
in swarm we are able to use this command but still pin to a specific version of docker will that not work here?